src/components/reusables/radialProgress/RadialProgress.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+
+export default class RadialProgress extends Component {
+  render() {
+    const { progress, strokeColor, strokeWidth, width } = this.props;
+		
+		// Determine the center of the circle, as that will be both the `cx` and the `cy` attribute.
+		const center = width / 2;
+		
+		// The stroke is applied half inside the circle, so we need to account for it in order to make it appear outside the circle.
+		const radius = (width / 2) - (strokeWidth / 2);
+		
+		// Set the circumference of the circle as our `stroke-dasharray` and our initial `stroke-dashoffset`
+		const strokeDasharray = 2 * radius * Math.PI;
+		
+		// Express progress as a percentage of stroke-dasharray. The difference between stroke-dasharray and this percentage is the stroke-dashoffset
+		const strokeDashoffset = progress > 0 ? (strokeDasharray - (strokeDasharray / 100 * progress)) : strokeDasharray;
+		
+		// We make a perfect circle, so it fits in a square.
+    const height = width;
+    
+    return (
+      <svg
```

Comment:
Try to reformat this svg

Reply:
To make the formating easy you can install prettier on your vscode

Reply:
I have prettier installed... A sample of the right formatting will help.